### ISSUE TITLE
can magnet be used with other taxonomy classifiers?

### ISSUE THREAD

**Issue by humbleflowers** [treangenlab/magnet#6]
magnet.py takes taxonomy identifier and abundances percentage as input. I was wondering if i can make a tsv with these columns from output of other tool and run this ? Are there are caveats or whether it was tested?

**Comment by beleafs** [comment]
Yes. You can use any taxonomy classifier as long as they include taxid in the output.
